node/cn/tracers/internal/tracers/assets.go

```diff
@@ -1,19 +1,19 @@
-// Code generated by go-bindata.
+// Code generated by go-bindata. DO NOT EDIT.
```

Comment:
May I ask how you generated this file?

Reply:
@jeongkyun-oh I installed go-bindata and run `go generate`
step 1. Install go-bindata: referred mac os installation of https://github.com/kevinburke/go-bindata
step 2. run `go generate`: `cd klaytn/node/cn/tracers/internal/tracers && go generate`
Please tell me if there's problem!

Reply:
Thanks for the description. Still, it would be great if you are able to check those tracers work or not.

Reply:
I checked if compatibility is maintained. With dev branch binary and current PR binary, I called debug api using every options(4bytetracer, etc.). I confirmed that they all worked the same :) Thanks for your advice!